common/src/app/common/logic/libraries.cljc

```diff
@@ -338,42 +339,54 @@
 (defn- generate-advance-nesting-level
   [changes file container libraries shape-id]
   (log/trace :msg "  -> advance-nesting-level")
-  (let [children  (cfh/get-children-with-self (:objects container) shape-id)
+  (let [detached-ids (transient #{})
```

Comment:
I think for what you're using the transient is better to have an `atom`

Reply:
ok